src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -906,6 +907,38 @@ public with sharing class CRLP_RollupUI_SVC {
         return fieldObjectList;
     }
 
+    /*******************************************************************************************************
+    * @description Finds writable, custom, unused target fields of specific types to provide field lists for Rollup definitions.
+    *              Includes an exception for a set summary field on the rollup detail.
+    * @param objectName the name of the object to look up
+    * @param selectedField the name of a summary field already in use
+    * @return List<Map<String, String>> an object with field api names, labels and data type.
+    */
+    private static Boolean fieldIsBanned(String objectName, String fieldName) {
```

Comment:
Is there a compelling reason for making a new method to do this, instead of doing it inline like we have with getFilterRuleFieldsExcludingTypes (eg: Set<String> excludedFields = new Set<String>{'ConnectionReceivedId', 'ConnectionSentId', 'IsDeleted', 'IsSplit', 'Fiscal', 'FiscalQuarter', 'FiscalYear', 'SyncedQuoteID'};)? You could create the map there too if the perf gains seem worth it. I wouldn't be opposed to similar mapping in getFilterRuleFieldsExcludingTypes too.